invenio_rdm_records/fixtures/users.py

```diff
@@ -47,11 +47,23 @@ def create(self, entry):
         """Load a single user."""
         email = entry.pop("email")
         password = self._get_password(email, entry)
+        username = entry.get("username")
```

Comment:
I think that `username` is a mandatory field when you register, can you check that?
If yes, I think that it must be added...